core/nodejsActionBase/src/service.js

```diff
@@ -114,6 +138,45 @@ function NodeActionService(config) {
         }
     };
 
+    this.webAction = function webAction(req) {
+        let body = req.body || {};
+        let params = req.params || {};
+        let actionName = params.action;
```

Comment:
this should include the package name too: 
```
	let pkg = params["package"] || "default";    
        let actionName = pkg + "/" + params.action; 
```

Reply:
actually we should we use this: 
```
	let pkg = params["package"] || "default";
	let actionName = pkg + "-" + params.action;    
        let fqActionName = pkg + "/" + params.action;
```

Reply:
fixed